app/components/Sidebar/components/Collections.js

```diff
@@ -1,98 +1,91 @@
 // @flow
-import { observer, inject } from "mobx-react";
+import fractionalIndex from "fractional-index";
+import { observer } from "mobx-react";
 import { PlusIcon } from "outline-icons";
 import * as React from "react";
-import { withTranslation, type TFunction } from "react-i18next";
-import keydown from "react-keydown";
-import { withRouter, type RouterHistory } from "react-router-dom";
-
-import CollectionsStore from "stores/CollectionsStore";
-import DocumentsStore from "stores/DocumentsStore";
-import PoliciesStore from "stores/PoliciesStore";
-import UiStore from "stores/UiStore";
+import { useDrop } from "react-dnd";
+import { useTranslation } from "react-i18next";
 import Fade from "components/Fade";
 import Flex from "components/Flex";
+import useStores from "../../../hooks/useStores";
 import CollectionLink from "./CollectionLink";
 import CollectionsLoading from "./CollectionsLoading";
+import DropCursor from "./DropCursor";
 import Header from "./Header";
 import SidebarLink from "./SidebarLink";
-import { newDocumentUrl } from "utils/routeHelpers";
-
 type Props = {
-  history: RouterHistory,
-  policies: PoliciesStore,
-  collections: CollectionsStore,
-  documents: DocumentsStore,
   onCreateCollection: () => void,
-  ui: UiStore,
-  t: TFunction,
 };
 
-@observer
-class Collections extends React.Component<Props> {
-  isPreloaded: boolean = !!this.props.collections.orderedData.length;
-
-  componentDidMount() {
-    const { collections } = this.props;
-
+function Collections({ onCreateCollection }: Props) {
+  const { ui, policies, documents, collections } = useStores();
+  const isPreloaded: boolean = !!collections.orderedData.length;
+  const { t } = useTranslation();
+  const orderedCollections = collections.orderedData;
+  React.useEffect(() => {
```

Comment:
```suggestion

  React.useEffect(() => {
```

Nit: A little extra space between methods and `const` would be nice :)